ci/Publish-MailMobileArtifacts.Jenkinsfile

```diff
@@ -0,0 +1,299 @@
+		stage("GitHub Release") {
+			stages {
+				stage("GitHub Android Tag") {
+					environment {
+						VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+						FILE_PATH = "build/app-android/tutanota-app-tutao-release-${VERSION}.apk"
+						GITHUB_RELEASE_PAGE = "https://github.com/tutao/tutanota/releases/tag/tutanota-android-release-${VERSION}"
+					}
+					when {
+						expression {
+							params.github && releaseNotes.android.trim()
+						}
+					}
+					steps {
+						script {
+							def util = load "ci/jenkins-lib/util.groovy"
+							util.downloadFromNexus(	groupId: "app",
+													artifactId: "android",
+													version: "${env.VERSION}",
+													outFile: "${env.WORKSPACE}/${env.FILE_PATH}",
+													fileExtension: 'apk')
+
+							if (!fileExists("${env.FILE_PATH}")) {
+								currentBuild.result = 'ABORTED'
+								error("Unable to find file ${env.FILE_PATH}")
+							}
+							echo "File ${env.FILE_PATH} found!"
+
+							catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS', message: 'Failed to upload android app to GitHub') {
+								writeReleaseNotes("android", "Android", "${env.VERSION}", "${env.WORKSPACE}/${env.FILE_PATH}")
+							}
+						} // script
+					} // steps
+				} // stage Android App
+				stage("GitHub iOS Tag") {
+					environment {
+						VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+						GITHUB_RELEASE_PAGE = "https://github.com/tutao/tutanota/releases/tag/tutanota-ios-release-${VERSION}"
+					}
+					when {
+						expression {
+							params.github && releaseNotes.ios
+						}
+					}
+					steps {
+						script {
+							catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS', message: 'Failed to upload iOS app to GitHub') {
+								writeReleaseNotes("ios", "iOS", "${env.VERSION}", "")
+							}
+						} // script
+					} // steps
+				} // stage iOS App
+			}
+		}
+		stage("Publishing Artifacts to Stores") {
+			parallel {
+				stage("Android App") {
+					environment {
+						VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+						FILE_PATH = "build/app-android/tutanota-app-tutao-release-${VERSION}.apk"
+						GITHUB_RELEASE_PAGE = "https://github.com/tutao/tutanota/releases/tag/tutanota-android-release-${VERSION}"
+					}
+					when {
+						expression {
+							params.googlePlayStore
+						}
+					}
+					steps {
+						script {
+							def util = load "ci/jenkins-lib/util.groovy"
+							util.downloadFromNexus(	groupId: "app",
+													artifactId: "android",
+													version: "${env.VERSION}",
+													outFile: "${env.WORKSPACE}/${env.FILE_PATH}",
+													fileExtension: 'apk')
+							if (!fileExists("${env.FILE_PATH}")) {
+								currentBuild.result = 'ABORTED'
+								error("Unable to find file ${env.FILE_PATH}")
+							}
+							echo "File ${env.FILE_PATH} found!"
+
+							catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS', message: 'Failed to upload android test app to Play Store') {
```

Comment:
I'm not sure it should be success, it would make sense for this job to fail if store upload fails